pubspec.yaml

```diff
@@ -39,3 +40,9 @@ dev_dependencies:
   http: '^0.11.0'
   js: '^0.6.0'
   scheduled_test: '^0.12.5'
+
+dependency_overrides:
```

Comment:
It's intended for now. I'll get rid of it once dart-lang/vm_service_client#13 lands.